portfolio/src/main/webapp/js/comments.js

```diff
@@ -57,18 +65,19 @@ function deleteComment(commentID) {
     params.append('action', 'delete');
     fetch('/comments', {method: 'POST', body: params}).then(response => {
         if (response.status === 200)
-            getComments(selectedSortType);
+            getComments(selectedSortType, selectedQuantityOfComments);
         else
             console.error(response.message);
     });
 }
 
-if (selectedSortType === 'date' || selectedSortType === 'rating')
-    getComments(selectedSortType);
-else {
-    getComments('date');
+if (selectedSortType !== 'date' && selectedSortType !== 'rating')
```

Comment:
From the [style guide](https://google.github.io/styleguide/jsguide.html#formatting-braces-all), if you are breaking the line after the `if`, you need to add the braces `{}`, even if it's a one line statement.